sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/HiveTableScanExec.scala

```diff
@@ -164,4 +164,28 @@ case class HiveTableScanExec(
   }
 
   override def output: Seq[Attribute] = attributes
+
+  override def sameResult(plan: SparkPlan): Boolean = plan match {
+    case other: HiveTableScanExec =>
+      val thisRequestedAttributes = requestedAttributes.map(cleanExpression)
+      val otherRequestedAttributes = other.requestedAttributes.map(cleanExpression)
```

Comment:
I only override the `sameReult` like `FileSourceScanExec `. I think `HiveTableScanExec` is used to text format and `FileSourceScanExec ` is used to parquet/orc format.